### PR TITLE
Publish android-persistent-sdk v1.30

### PR DESCRIPTION
**Pull request by julo15**
@dgranahan @seikojin @rudmer90

This PR publishes out v1.30 to affiliates.

#### Changes
* Apache `HttpClient` has been replaced with Retrofit
* Bump tool versions in SampleApp

#### Testing
* Looking at syncs from dogfooders. So far things are looking good. Will wrap this up tomorrow
* Will wait for thumbs up from QA

### CODE REVIEW

File: SampleApp/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,19 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.placed.android.sampleapp">
 
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```
Comment (rudmer90) [inline SampleApp/app/src/main/AndroidManifest.xml:5]: Why were these removed?

File: SampleApp/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -25,16 +18,6 @@
         </activity>
 
         <meta-data android:name="placed_app_key" android:value="@string/placed_api_key" />
-        <service android:name="com.placed.client.android.persistent.PlacedService"/>
```
Comment (rudmer90) [inline SampleApp/app/src/main/AndroidManifest.xml:28]: Why were these removed also?

**Review by rudmer90 — APPROVED** [review]

File: SampleApp/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (julo15) [reply SampleApp/app/src/main/AndroidManifest.xml:5]: These are in the SDK AAR now :). Ditto for below
